### DISCUSSION TITLE
Mistake in verification

### DISCUSSION THREAD

**Discussion by FaridKaveh** [community/community#109722] [GitHub Education]
### Hi everyone,

I'm actually having two problems here. 

1. While trying to verify myself as a student on GitHub Education, I accidentally uploaded an image of an enrollment letter for an earlier degree, so the date on it would not show that I am a student today. Despite this the application has (apparently) been successful (which is strange in itself). However, I have not received the education benefits OR any sort of confirmation email from GitHub. 

2. I actually had previously verified my student status back in 2021, and that has not expired. But now that I think of it, I realise that I never received any GitHub Education benefits after that application either. This is what my education status looks like on https://education.github.com/discount_requests/application

![Screenshot from 2024-02-23 17-45-59](https://github.com/community/community/assets/77326393/80cf6d96-d3ea-4b66-859f-9dcf3367f1d2)

I was thinking that maybe this problem could be fixed if I could be made unverified, and then I verified again properly this time. I wonder if GitHub thinks that I already have student benefits from 2021. 

Anyway to make myself 'unverified'? Or any other solutions?

**Comment by namith1003** [comment]
No, many people including myself have been confused regarding this, the approved status is not approved it is more like pending, you could still be rejected in case the document is not accepted, and you will have to wait patiently, people have reported that full approval takes between 2 to even more than 8 days. "It says your benefits will be available soon" and if you go to the 

https://education.github.com/pack

page it states that the application is still under process. You will have to wait patiently to see the result.

Hope this answers your question.
